api.go

```diff
@@ -52,7 +52,7 @@ func main() {
 		log.Fatalf("Cannot parse duration %s. Got: %v", config.DefaultSessionDuration, err)
 	}
 
-	playground := types.Playground{Domain: config.PlaygroundDomain, DefaultDinDInstanceImage: config.DefaultDinDImage, AllowWindowsInstances: config.NoWindows, DefaultSessionDuration: d, AvailableDinDInstanceImages: []string{config.DefaultDinDImage}}
+	playground := types.Playground{Domain: config.PlaygroundDomain, DefaultDinDInstanceImage: config.DefaultDinDImage, AllowWindowsInstances: config.NoWindows, DefaultSessionDuration: d, AvailableDinDInstanceImages: []string{config.DefaultDinDImage}, Tasks: []string{".*"}}
```

Comment:
Didn't we say that the default should be  "don't run any tasks unless specified" ?

Reply:
The default of the `Playground` struct is an empty string slice, which means don't run anything. But when you run the playground locally (for development for example) you want the scheduler to do something, right? Like show the ports that you open, etc. Do you prefer it not to show anything and change the current behavior?

Reply:
But if the config will be retrieved from the DB shoudn't we populate the local db with that config instead of hardcoding it in the code?

Reply:
How do you populate the DB with the playground for development if you don't do it here?